### DISCUSSION TITLE
Select parameters (#datapoints/iterations) before starting analysis

### DISCUSSION THREAD

**Discussion by sitangshugk95** [ContactEngineering/topobank#883] [Q&A] [answered]
I might have put an additional '0' at the end of the # datapoints (100 instead of 10), and the analysis is taking forever to run.  Every new analysis that I am running is taking long (presumably because it has saved the '100' from the previously chosen option.
Is there a way I can check (and change) what the analysis parameters are set to, BEFORE starting the analysis?

**Comment by mcrot** [comment]
Every time you start a new contact analysis, you see the parameters in the *Parameters* section:
![grafik](https://user-images.githubusercontent.com/9113541/183752456-3cf3912b-b826-4e9c-9426-0986cbdd8f99.png)

If you look at an existing analysis, and if the parameters are the same for all selected measurements (also see #885 ),
these common parameters are shown in the boxes, also the "Number of steps". Probably you mean that with # datapoints.

Do you see 100 there? Could you enter 10 instead of 100 before pressing *Recalculate*?

**Reply by sitangshugk95** [reply]
You're right, I meant # steps as # datapoints.

The issue is that it is stuck in "Pending" forever. 
![Pending](https://user-images.githubusercontent.com/108830413/183755483-3354db7f-0e5d-4d5b-aeef-871d6250941f.jpg)
and the task information options do not show up when I click on "Open" 
![after clicking Open](https://user-images.githubusercontent.com/108830413/183755542-4ceedcac-81d9-4864-973e-492bb5de1b87.jpg)
.

**Comment by mcrot** [comment, marked as answer]
I've restarted that analysis. Does it show up now? I think maybe we need some buttons to retrigger/stop analyses in the web app.

**Reply by sitangshugk95** [reply]
Yes, this works now, thank you.

I agree, adding the button to retrigger/stop would be a good idea.
